python/tutorials/03-matrix-multiplication.py

```diff
@@ -196,6 +205,22 @@
                       num_warps=4),
         triton.Config({'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 32, 'BLOCK_SIZE_K': 64, 'GROUP_SIZE_M': 8}, num_stages=4,
                       num_warps=4),
+    ] if not is_hip() else [
```

Comment:
This is a huge ` <a> if <condition> else <b>`. I think conditional expression is more meant to be as oneliner. Maybe structure it in the following way to be clear:

```python

def get_cuda_autotune_config(): ...
def get_hip_autotune_config(): ...
def get_autotune_config(): 
  target = triton.runtime.driver.active.get_current_target()
  if target.backend == "cuda": ...
  if ...

@triton.autotune(config=get_autotune_config() ...)
```